src/appleseed/renderer/modeling/aov/aovfactoryregistrar.h

```diff
@@ -84,6 +95,9 @@ class APPLESEED_DLLSYMBOL AOVFactoryRegistrar
     struct Impl;
     Impl* impl;
 
+    // Pointer to the loaded libraries in memory.
+    boost::shared_ptr<loaded_libs_container> m_loaded_libraries;
```

Comment:
I wonder if we need to store loaded libraries in each registrar.

Reply:
I don't think we need, but I thought it will be good for a future work, although reinitialize is called in the constructor of each plugin type which doesn't make sense after the changes I have made, 
so probably until now we needn't to store loaded libraries